reframe/core/pipeline.py

```diff
@@ -1160,11 +1160,8 @@ def cleanup(self, remove_files=False, unload_env=True):
             self._current_partition.local_env.unload()
 
     def __str__(self):
-        return ('%s (found in %s)\n'
-                '        descr: %s\n'
-                '        tags: {%s}, maintainers: %s' %
-                (self.name, inspect.getfile(type(self)),
-                 self.descr, ','.join(self.tags), self.maintainers))
+        return ("%s(name='%s', prefix='%s')" %
+                (type(self).__name__, self.name, self.prefix))
```

Comment:
It's against the coding style to put parentheses around the return value, unless it is absolutely necessary. Can you rewrite this as I proposed before?